### DISCUSSION TITLE
Cockpit on apache2

### DISCUSSION THREAD

**Discussion by Ludo-code** [cockpit-project/cockpit#18152] [Q&A] [answered]
Page: 

It would be nice to use cockpit directly in apache2 for all needed web control panel interface... Because only webmin have at this time a apache support...
Can you take cockpit on apache2?

**Comment by martinpitt** [comment, marked as answer]
Yes, you can run Apache (or nginx) as reverse proxy for cockpit. See e.g. https://github.com/cockpit-project/cockpit/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt
